packages/stark-core/src/modules/error-handler/handlers/stark-error-handler.ts

```diff
@@ -0,0 +1,52 @@
+import {ErrorHandler, Injectable, Injector} from "@angular/core";
+import {Store} from "@ngrx/store";
+import {StarkCoreApplicationState} from "../../../common/store";
+import {StarkUnhandledError} from "../actions";
+import {STARK_LOGGING_SERVICE, StarkLoggingService} from "../../logging/services";
+
+
```

Comment:
We should remove the `stark-` prefix from the name of this file... it is already implicit since it is part of the Stark packages ;)   We should just keep the prefix in the name of the class